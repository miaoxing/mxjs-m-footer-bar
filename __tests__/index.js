import FooterBar from '..';
import {render} from '@testing-library/react';

describe('FooterBar', () => {
  test('basic', () => {
    const {container} = render(<FooterBar>test</FooterBar>);
    expect(container).toMatchSnapshot();
  });

  test('FooterBar.Icon', () => {
    const {container} = render(<FooterBar>
      <FooterBar.Icon>
        test
      </FooterBar.Icon>
    </FooterBar>);
    expect(container).toMatchSnapshot();
  });

  test('FooterBar.Button', () => {
    const {container} = render(<FooterBar>
      <FooterBar.Button>
        test
      </FooterBar.Button>
    </FooterBar>);
    expect(container).toMatchSnapshot();
  });
});
