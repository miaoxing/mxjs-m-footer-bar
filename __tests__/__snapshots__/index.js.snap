// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBar FooterBar.Button 1`] = `
<div>
  <taro-view-core
    class="mx-footer-bar __mx-footer-bar-1"
  >
    <taro-button-core
      class="mx-btn mx-footer-bar-btn"
    >
      test
    </taro-button-core>
  </taro-view-core>
</div>
`;

exports[`FooterBar FooterBar.Icon 1`] = `
<div>
  <taro-view-core
    class="mx-footer-bar __mx-footer-bar-1"
  >
    <taro-button-core
      class="mx-btn mx-footer-bar-icon"
    >
      test
    </taro-button-core>
  </taro-view-core>
</div>
`;

exports[`FooterBar basic 1`] = `
<div>
  <taro-view-core
    class="mx-footer-bar __mx-footer-bar-1"
  >
    test
  </taro-view-core>
</div>
`;
